<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamCombineClass</class>
 <widget class="QWidget" name="StreamCombineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamCombine</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="SerialBox">
     <property name="title">
      <string>SerialPort</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TcpBox">
     <property name="title">
      <string>TcpClient</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="NtripBox">
     <property name="title">
      <string>NtripClient</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
